<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>enigmaClass</class>
 <widget class="QMainWindow" name="enigmaClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>enigma</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>581</width>
      <height>381</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabEncode">
     <attribute name="title">
      <string>编码</string>
     </attribute>
     <widget class="QLineEdit" name="lineTxtNowAddress">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>261</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>当前路径</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnEncode">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>40</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>编码当前路径下的文本</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnEncodeChooseAddress">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>40</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>选择路径</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbtnEncodeSaveSide">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>编码图片保存到文本旁</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbtnEncodeChoose">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>80</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>选择保存路径</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEncodeSaveNowAddress">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>80</y>
        <width>210</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>当前路径</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chkEncodeDebug">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>320</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>开启详细调试信息</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblEncodeState">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>110</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>信息统计</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="txtEncodeDebug">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>540</width>
        <height>170</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabDecode">
     <attribute name="title">
      <string>解码</string>
     </attribute>
     <widget class="QLineEdit" name="lineBmpNowAddress">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>261</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>当前路径</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDecodeChooseAddress">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>40</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>选择路径</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDecode">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>40</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>解码当前路径下二维码</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbtnDecodeSaveSide">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>解码文本保存到图片旁</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbtnDecodeChoose">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>80</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>选择保存路径</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineDecodeSaveNowAddress">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>80</y>
        <width>210</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>当前路径</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chkDecodeDebug">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>320</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>开启详细调试信息</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>信息统计</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblDecodeState">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>110</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTextBrowser" name="txtDecodeDebug">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>540</width>
        <height>170</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="enigma.qrc"/>
 </resources>
 <connections/>
</ui>
